lab/spending_guidelines.md

```diff
@@ -0,0 +1,83 @@
+# OPI Lab Budget Guidelines
+
+This document provides guidance for using OPI funds to acquire lab
+equipment.  We describe both "appropriate" and "inappropriate" use of
+equipment funds and include examples of each.  Generally, commonly
```

Comment:
To be clear your intention is that the below is a set of examples not an exhaustive list of the appropriate equipment?

Reply:
Sounds reasonable to me.

Reply:
Yes.  Two parts here.  First, that is why file name has "guideline".  Secondly, expectation is folks should issue additional PRs to augment this list.  I expect some neutral "equipment czar" to be designated by the TSC to have final say on purchase requests.  That person should exercise judgment to approve at least large purchase requests.